frontend/public/components/container.jsx

```diff
@@ -108,6 +108,21 @@ const Env = ({env}) => {
   </table>;
 };
 
+// Split image string into the Docker image string and tag (aka version) portions. The tag defaults to 'latest'.
+const getContainerImageAndTag = image => {
+  if (_.isEmpty(image)) {
+    return [null, null];
+  }
+  const semiColonSplit = image.split(':');
+  if (_.size(semiColonSplit) === 1) {
+    return [image, 'latest'];
+  }
+  const imageTag = _.last(semiColonSplit);
+  const imageNamePortion = _.trim(semiColonSplit.slice(-2, -1), '@sha256');
+  const imageName = imageNamePortion.split('/').slice(-2).join('/');
+  return [imageName, imageTag];
```

Comment:
Suggest we have special handling for the `@sha256` case then use `image.lastIndexOf(':')` in other cases to get the tag.

Would be good to have examples and their output there. I'm not sure exactly how split/slice/join is changing the string.

Reply:
I know the previous code had an array since it used split, but I'd prefer to return an object here with named properties like 

```js
return { imageName, imageTag };
```